lib/tasks/globus.rake

```diff
@@ -3,14 +3,14 @@
 namespace :globus do
   desc 'Delete Globus access rules and empty directories'
   task :prune_access, [:sunet_id] => :environment do |_task, args|
-    globus_destinations = GlobusDestination.joins(:user).where('user.sunet_id': args[:sunet_id])
-    puts "No GlobusDestinations found for #{args[:sunet_id]}" if globus_destinations.empty?
+    sunet_id = args[:sunet_id]
+    globus_destinations = GlobusDestination.where(user: User.find_by(sunet_id:))
+    puts "No GlobusDestinations found for #{sunet_id}" if globus_destinations.empty?
     globus_destinations.each do |dest|
```

Comment:
refactor of the query to make it use built in ActiveRecord joining syntax

Reply:
This doesn't feel more idiomatic. Perhaps:
```
GlobusDestination.joins(:user).where(user: {sunet_id: args[:sunet_id]})
```

Reply:
yeah, ok -- i guess this saves an extra query against the User model

Just a side note that it seems odd to me that the `GlobusDestination` belongs to both `User` and `BatchContext`, since the `BatchContext` is directly linked with a user (i.e. you could always reach the user through the `BatchContext` even without connecting the `GlobusDestination` to user in the model, and if they get out of sync it might cause weird behavior).